### DISCUSSION TITLE
waiting with nfs server start, volume is not attached

### DISCUSSION THREAD

**Discussion by rajeevpadinharepattu** [longhorn/longhorn#4755] [General]
I have a single node cluster, where i have multiple longhorn volume 
2 of them are attached to multiple pods.  (RWX), the cluster was working fine till yesterday

Suddenly one of the pod started getting the below error and pods cannot get attached to the volumes, I have already done the multipath configuration, also completed all the pre-requisite to run longhorn on a cluster ( like nfs4, iscsi packages)

all pods except below up and running

share-manager-pvc-826e159f-ec6e-420f-a781-55305a348e10   1/1     Running   0               3h58m
share-manager-pvc-9660a183-5cf4-4d74-ab58-2a763eb91bff   0/1     Running   0                3h57m

and if i investigate the share manager logs, I get the below errors back-to-back

time="2022-10-20T14:16:54Z" level=warning msg="waiting with nfs server start, volume is not attached" encrypted=true volume=pvc-9660a183-5cf4-4d74-ab58-2a763eb91bff

I can see the below error in longhorn manager pod

time="2022-10-20T14:23:39Z" level=warning msg="Dropping Longhorn volume longhorn-system/pvc-9660a183-5cf4-4d74-ab58-2a763eb91bff out of the queue" controller=longhorn-volume error="fail to sync longhorn-system/pvc-9660a183-5cf4-4d74-ab58-2a763eb91bff: fail to reconcile volume state for pvc-9660a183-5cf4-4d74-ab58-2a763eb91bff: no healthy or scheduled replica for starting" node=rtx5000

somebody could help to troubleshoot the issue? even after the cluster restart, the issue could not resolved.

kubernetes version: 1.22
longhorn version: 1.24

the support bundle is attached.

[longhorn-support-bundle_c34ae433-06f2-4ddd-8cb1-3a099c2fcbb0_2022-10-20T06-23-41Z.zip](https://github.com/longhorn/longhorn/files/9831076/longhorn-support-bundle_c34ae433-06f2-4ddd-8cb1-3a099c2fcbb0_2022-10-20T06-23-41Z.zip)

**Comment by PhanLe1010** [comment]
Looked into the support bundle. The volume `pvc-9660a183-5cf4-4d74-ab58-2a763eb91bff` was crash on 10/20 (hardware issue, etc,..) as indicated by `remountrequestedat: "2022-10-20T05:05:24Z"`. This volume only has 1 replica so it become faulted. Solution: 
1. Scale down all workload that is using this volume
1. Go to Longhorn UI -> click on the volume tab -> go to volume detail page. Click on the menu bar on top right -> select salvage volume -> select and salvage the only replica
1. Scale up the workload

**Reply by rajeevpadinharepattu** [reply]
@PhanLe1010, Thanks for the update, i had to recreate the volumes as it was quite urgent, I will try this solution nexttime when we get this error.
